lib/linux.js

```diff
@@ -41,18 +41,23 @@ var LinuxBuilder = {
       linux.archName = 'x86_64';
     }
 
-    var tmpFolder = path.join( os.tmpDir(), linux.executable );
+    var outFilename = linux.executable + '-' +
```

Comment:
It will be better just use standard, as electron-packager does. But, of course, I don't insist and don't want to start religious war. It is ok for me to run `npm test` before commit and fix all style issues.